cmd/controller/main.go

```diff
@@ -180,7 +181,8 @@ type controllerRunOptions struct {
 	node machinecontroller.NodeSettings
 }
 
-func main() {
+// parseFlags encapsulates the parsing of all flags.
+func parseFlags() {
```

Comment:
Why?

Reply:
After adding only my one flag the cyclomatic complexity test failed. 🤦‍♂ Moving it out into this function made it pass again. 

IMHO the better solution would be a refactoring into a `Main` type with helping methods and a major `Run()` method called in `main()`. This would also be possible to be unit tested (needing a lot of stubbing). But that would be a different issue.

Reply:
Or just remove the cyclomatic complecity linter? It doesn't care about nesting at all it just counts number of branches within a function which IMHO is a pretty useless metric.

Reply:
Would have no problem with it. Got some similar messages in my private code too where IMHO the splitting with the needed data structures and/or arguments would lead to an even more unmaintainable code.

But we need a general agreement then.

Reply:
Can you just add an exception for this specific case so we can get this pull merged?

Reply:
Done.